CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## master / unreleased
 
+* [ENHANCEMENT] Upgraded Prometheus to 2.12.0 and Alertmanager to 0.19.0 #1597
+  * [CHANGE] No longer supports auto Slack root, hence `--alertmanager.configs.auto-slack-root` flag was also dropped.
```

Comment:
Change comes before Enhancement.

Reply:
I had actually indented it as it was part of the same PR. I will make those as 2 entries instead.

Reply:
Modified